registry/client/repository.go

```diff
@@ -292,9 +292,22 @@ func (t *tags) Get(ctx context.Context, tag string) (distribution.Descriptor, er
 	if err != nil {
 		return distribution.Descriptor{}, err
 	}
+
+	req, err := http.NewRequest("HEAD", u, nil)
+	if err != nil {
+		return distribution.Descriptor{}, err
+	}
+
+	for _, t := range distribution.ManifestMediaTypes() {
+		req.Header.Add("Accept", t)
+	}
+
 	var attempts int
-	resp, err := t.client.Head(u)
 
+	resp, err := t.client.Do(req)
```

Comment:
Doesn't the GET request on like 320 also need to be changed to include the Accept headers?